src/ResourceManagement/HDInsight/HDInsight.Tests/UnitTests/ClusterCreateTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿using HDInsight.Tests.Helpers;
+using Microsoft.Azure.Management.HDInsight;
+using Microsoft.Azure.Test;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.Azure.Management.HDInsight.Models;
+using Microsoft.Azure.Management.Resources;
+using System.Threading;
+using Newtonsoft.Json;
+using System.Collections;
+
+namespace HDInsight.Tests.UnitTests
+{
+    public class ClusterCreateTests
+    {
+        private Mock<HDInsightManagementClient> clientMock;
+        private Mock<ClusterOperations> clusterOperationsMock;
+
+        public ClusterCreateTests()
+        {
+            clientMock = new Mock<HDInsightManagementClient>();
+            clusterOperationsMock = new Mock<ClusterOperations>(clientMock.Object);
+
+            clientMock.SetupGet(c => c.Clusters).Returns(clusterOperationsMock.Object);
+
```

Comment:
This was already fixed in https://github.com/Azure/azure-sdk-for-net/pull/2484/commits/056de19002d77dd18782e7c7c973c88a6ff83cba.
I am not sure where you are getting this from.